cerner/terra-dev-site
# Update props table source error

### Summary
Changed the error message in devSitePropsTable.js to be more informative as opposed to the previous code error block that was displayed when the props table could not find a source.

Closes #252 

### Testing
1. Compile the terra-dev-site project
2. Change the name of the [source file](https://github.com/cerner/terra-dev-site/blob/master/src/terra-dev-site/test/loaders/PropsTableExample.jsx), ex: `PropsTableExample.jsx` -> `PropsTableExample2.jsx`
3. Enter `npm run compile:prod` in the terminal
4. The updated error should appear as it does below and the project should be unable to compile.

```
ERROR in ./lib/terra-dev-site/test/loaders/PropsTableExample.js?dev-site-props-table
Module build failed (from ./config/webpack/loaders/devSitePropsTable.js):
Error: A non transpiled source file is required for the props table to be generated:
/Users/dm068655/Documents/terra-dev-site/src/terra-dev-site/test/loaders/PropsTableExample
```
Thank you for contributing to Terra.
@cerner/terra
